js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -1,7 +1,7 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var ExtensionStore = require('./ExtensionStore.js').instance;
-var ResourceLoadTracker = require('./ResourceLoadTracker').instance;
+var importedExtensionStore = require('./ExtensionStore.js').instance;
+var importedResourceLoadTracker = require('./ResourceLoadTracker').instance;
```

Comment:
I've been thinking about this too. I wonder if we could shift toward individual files simply exporting a func or class, and letting another file ("index"?) be responsible for creating instances and wiring things together. This would give us some flexibility in tests to set things up with mocks. The Capability API in blueocean-core-js does this, and it really helped from a testability perspective.